packages/rocketchat-api/server/v1/subscriptions.js

```diff
@@ -25,6 +25,28 @@ RocketChat.API.v1.addRoute('subscriptions.get', { authRequired: true }, {
 	}
 });
 
+RocketChat.API.v1.addRoute('subscriptions.getOne', { authRequired: true }, {
```

Comment:
I know this is just for renaming endpoints, but since I was not able to comment on the old PR I'll do it now =)

what is the difference between `subscriptions.getOne` and `subscriptions.get`? IMO they should be the same accepting different queries.

Reply:
One calls the method and one calls directly from the database. I haven't had a chance to look into what exactly that method does, so no clue if there is a difference.

Reply:
@sampaiodiego sadly, I'm going to have to say that we should keep the calls separated. The main reason for this is because the `subscriptions.get` returns an object with two properties that are arrays. If we switch to allow support for getting only one, either we would have to change the response format for it to make sense or use the current format which doesn't make sense when returning/getting only one. So, keeping the `subscriptions.getOne` makes sense for the data being returned. What's your thoughts?

Reply:
yep.. I've tested this and saw the payload differences as well.. 😞 agreed keeping both separated